src/main/java/step1/StringAddCalculator.java

```diff
@@ -0,0 +1,47 @@
+package step1;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringAddCalculator {
+    static public int splitAndSum(String text) {
+        if (text == null || text.isEmpty()) return 0;
```

Comment:
```suggestion
        if (text == null || text.isEmpty()) {
                return 0;
        }
```

자바의 경우 하나의 조건문에도 괄호를 작성하는 것이 관례입니다!
컨벤션은 https://google.github.io/styleguide/javaguide.html 또는 https://myeonguni.tistory.com/1596 를 참고해주세요~